projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -578,7 +580,15 @@ public function jetpack_user_table_row_actions( $actions, $user_object ) {
 				);
 			}
 
-			unset( $actions['resetpassword'] );
+			if (
+				current_user_can( 'promote_users' )
+				&& (
+					$has_pending_invite
+					|| Jetpack::connection()->is_user_connected( $user_id )
```

Comment:
Noting that this could then be improved with the new utility introduced in #36572.